docs/topics/http/urls.txt

```diff
@@ -597,16 +597,34 @@ In order to perform URL reversing, you'll need to use **named URL patterns**
 as done in the examples above. The string used for the URL name can contain any
 characters you like. You are not restricted to valid Python names.
 
-When you name your URL patterns, make sure you use names that are unlikely
-to clash with any other application's choice of names. If you call your URL
-pattern ``comment``, and another application does the same thing, there's
-no guarantee which URL will be inserted into your template when you use
-this name.
+First, when you name your own URL patterns, make sure you use names that are
+unlikely to clash with any other application's choice of names. If you call
+your URL pattern ``comment``, and another application does the same thing,
+there's no guarantee which URL will be inserted into your template when you
+use this name, it will depend on the order of .
 
 Putting a prefix on your URL names, perhaps derived from the application
 name, will decrease the chances of collision. We recommend something like
 ``myapp-comment`` instead of ``comment``.
 
+Second, you can deliberately use the *same name* as used in another (built-in)
```

Comment:
I assume by "built-in" application you mean contrib apps, but doesn't the same principle apply to any third-party app?